prow/github/client.go

```diff
@@ -2969,6 +2970,35 @@ func (c *client) DeleteRef(org, repo, ref string) error {
 	return err
 }
 
+// ListFileCommits returns the commits for this file path.
+//
+// See https://developer.github.com/v3/repos/#list-commits
+func (c *client) ListFileCommits(org, repo, filePath string) ([]RepositoryCommit, error) {
```

Comment:
Would be nice to throw in a quick test for this, the GetRef test is probably a nice framework for it

Reply:
Added.